test/code/providers/Container_HostInventory_Class_Provider_UnitTest.cpp

```diff
@@ -0,0 +1,191 @@
+#include <set>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <unistd.h>
+#include <uuid/uuid.h>
+#include <vector>
+
+#include <scxcorelib/scxcmn.h>
+#include <scxcorelib/scxprocess.h>
+#include <scxcorelib/stringaid.h>
+#include <testutils/scxunit.h>
+#include <testutils/providertestutils.h>
+
+#include "Container_HostInventory_Class_Provider.h"
+#include "cjson/cJSON.h"
+
+using namespace std;
+using namespace SCXCoreLib;
+
+class ContainerHostInventoryTest : public CppUnit::TestFixture
+{
+    CPPUNIT_TEST_SUITE(ContainerHostInventoryTest);
+    CPPUNIT_TEST(TestK8Parsing);
+    CPPUNIT_TEST(TestDcosParsing);
+    CPPUNIT_TEST(TestSwarmModeParsing);
+    CPPUNIT_TEST(TestSwarmParsing);
+    CPPUNIT_TEST(TestNonOrchestratedParsing);
+    //Running this test at the end to test that the unset of DOCKER_TESTRUNNER_STRING is successful
+    CPPUNIT_TEST(TestEnumerateInstances);
+    CPPUNIT_TEST_SUITE_END();
+private:
+
+public:
+    
+protected:
+    void TestEnumerateInstances()
+    {
+        wstring errMsg;
+        TestableContext context;
+        vector<wstring> m_keyNames;
+        m_keyNames.push_back(L"InstanceID");
+
+        // Enumerate provider. This just tests if the provider is able to pick up properties from a docker info call on the machine
+        StandardTestEnumerateInstances<mi::Container_HostInventory_Class_Provider>(m_keyNames, context, CALL_LOCATION(errMsg));
+
+        CPPUNIT_ASSERT(context.Size());
```

Comment:
Please do an ASSERT_EQ that the size is 1, not some non-zero size.

Change all other tests that do this as well.